apps/web/pages/api/auth/reset-password.ts

```diff
@@ -42,14 +42,16 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       },
       data: {
         password: {
-          update: {
-            hash: hashedPassword,
+          upsert: {
+            create: { hash: hashedPassword },
+            update: { hash: hashedPassword },
           },
         },
         emailVerified: new Date(),
       },
     });
   } catch (e) {
+    console.error("Error updating password", e);
```

Comment:
the error is not necessarily related to updating the password, could also fail if the email is not found